client/stylesheets/_toCat.scss

```diff
@@ -5184,16 +5184,66 @@ On click the images will be displayed at normal size to complete the effect
 
 .headerSearchRadiusLogo {
     margin-right: 30px;
-    font-weight: 700;
+    font-weight: 400;
 }
 
 .headerSearchRadiusLogo a {
 	font-size: 1.3rem !important;
 }
 
+.headerSearchNavLogo {
+	margin-left: 16px;
+	margin-top: 3px;
+}
+
 /* End */
 
+/* Nav Search */
 
+.headerSearchNavField {
+	color: #404040;
+    font-weight: 400;
+    font-size: 0.9rem !important;
+}
+
+.headerSearchNavField i {
+	color: #404040 !important;
+	border-bottom: 2px solid #404040;
+    padding: 0 5px 6px 0;
+    margin-right: -5px;
+}
+
+.headerSearchNav {
+	outline: none !important;
+	-webkit-appearance: none !important;
+    -moz-appearance: none !important;
+    appearance: none !important;
+	background-color: #f8504b;
+    border: none;
+    border-bottom: 2px solid #404040 !important;
+    letter-spacing: 1px;
+    padding-bottom: 5px;
+    font-family: 'Droid Sans', sans-serif !important;
+    margin-top: 5px;
+}
+
+.headerSearchNav::-webkit-input-placeholder {
+	color: #333 !important;
+}
+
+.headerSearchNav::-moz-placeholder {
+	color: #333 !important;
+}
+
+.headerSearchNav:-mozplaceholder {
+	color: #333 !important;
```

Comment:
Color literals like `#333` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>!important should not be used<br>Line should be indented with spaces, not tabs